src/scheduler/fifo.c

```diff
@@ -2766,6 +2764,8 @@ validate_sched_attrs(int connector)
 	ss = bs_find(all_ss, sc_name);
 
 	if (ss == NULL) {
+		snprintf(log_buffer, sizeof(log_buffer), "Unable to retrieve the scheduler attributes from server");
+		log_err(-1, __func__, log_buffer);
```

Comment:
This is a static string, just put it into log_err itself.

Reply:
oops! I merged this PR already, I didn't realize that there was a commit that you hadn't reviewed yet, sorry about that!

Reply:
@bhroam, I am ready to implement this comment but I don't know whether I can since the PR is already merged. Since having a variable length string does not harm I am thinking to leave this as is for now and implement this comment in upcoming PR. Hope this is fine.

Reply:
It doesn't harm anything, but this was a common thing you did when you implemented multi-sched.  I just went through and fixed them all to call log_event() directly.  I'd hate to add another.  I guess we'll fix it later.